targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F412xG/system_clock.c

```diff
@@ -88,7 +88,7 @@ void SetSysClock(void)
 /******************************************************************************/
 /*            PLL (clocked by HSE) used as System clock source                */
 /******************************************************************************/
-uint8_t SetSysClock_PLL_HSE(uint8_t bypass)
+__weak uint8_t SetSysClock_PLL_HSE(uint8_t bypass)
```

Comment:
Not needed, as SetSysClock is weak

Reply:
Actually quite the opposite, SetSysClock does not change for targets but SetSysClock_PLL_HSE and SetSysClock_PLL_HSI implementations change. I think these two should be weak and SetSysClock not.

Reply:
Sounds OK